pkg/protocol/request.go

```diff
@@ -450,6 +450,20 @@ func (req *Request) SetQueryString(queryString string) {
 	req.URI().SetQueryString(queryString)
 }
 
+// SetQueryParams sets query Params.
+func (req *Request) SetQueryParams(params map[string]string) {
+	var builder strings.Builder
+	builder.Write([]byte("?"))
```

Comment:
No need to add "?", check here: https://github.com/cloudwego/hertz/blob/develop/pkg/protocol/uri.go#L653

Reply:
Maybe add the correctness check in UTs.